src/app.ts

```diff
@@ -953,11 +954,13 @@ function registerZendeskRoutes(
   profileService: ProfileService,
   tokenService: TokenService,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  bearerZendeskTokenAuth: any
+  bearerZendeskTokenAuth: any,
+  appInsightsClient?: appInsights.TelemetryClient
 ): void {
   const zendeskController: ZendeskController = new ZendeskController(
     profileService,
-    tokenService
+    tokenService,
+    appInsightsClient
```

Comment:
Can we pilot this tracking with a FF? Se we can enable and disable it whitout executing a new deployment.

Reply:
Given that it should never occur, and considering its rarity, is it possible for us to leave without any FF?